automated install/basic-install.sh

```diff
@@ -1885,7 +1885,81 @@ create_pihole_user() {
     fi
 }
 
-#
+# replaceAddCreateRename takes three varialbles
+# 1 - the name of the local variable that represents the new value
+# 2 - the name of the variable in the target file
+# 3 - the path and name of the target file to modify
+replaceAddCreateRename() {
```

Comment:
maybe a better name for this function would be something like `saveSetting` or the slightly more verbose`saveSettingToFile`

Reply:
I agree, it needs a better name.  I could not think of anything good so I just put what it did.  It can be used for more than just saving setting to a file though.  It's main purpose is to create and/or update variable files.

How about updateVarsFile (I've been working with Ansible a lot lately, it uses vars_files for everything.)

Reply:
Also, there are two different functions, replaceAddCreateRename and replaceAddCreate.  Both have the same purpose but they take different arguments.

replaceAddCreate only has two arguments, the name of the variable and the file to modify.  It assumes the name of the variable in the file is the same as the variable in the basic-install script.

replaceAddCreateRename has three arguments, the name of the variable in the basic-install script, the name of the varialbe in the vars file, and the file to modify.

I will remove 'replaceAddCreate' completely and then just modify the one time it's called to use 'replaceAddCreateRename', renamed to updateVarsFile